### PR TITLE
Add links to Install and Tuturials

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -16,11 +16,13 @@
     <div class="wrapper">
       <header>
         <h1 class="header"><a href="https://buildah.io">Buildah</a></h1>
-        <p class="header">Buildah - a tool that facilitates building OCI images.</p>
+        <p class="header">Buildah - a tool that facilitates building OCI container images.</p>
```
Comment (TomSweeneyRedHat) [inline _layouts/default.html]: This line came from the top of the Buildah GitHub page (https://github.com/containers/buildah).  I don't see a way to change it from my side, can you touch it up there too?

File: _layouts/default.html [hunk 2]
```diff
@@ -16,11 +16,13 @@
...
 
         <ul>
           <li><a class="buttons" href="{{ site.baseurl }}/blogs">Blogs</a></h2>
           <li><a class="buttons" href="{{ site.baseurl }}/releases">Releases</a>
+          <li><a class="buttons github" href="https://github.com/containers/buildah/blob/master/install.md">Install</a>
+          <li><a class="buttons github" href="https://github.com/containers/buildah/tree/master/docs/tutorials">Tuturials</a>
```
Comment (TomSweeneyRedHat) [inline _layouts/default.html]: No British English card here me thinks.  Tuturials to Tutorials.

**Comment by TomSweeneyRedHat** [comment]
@sanjabonic I think this is OK, but I'm always leery about changes to columns like this.  Is there a way I could preview changes like this before merging?

**Comment by TomSweeneyRedHat** [comment]
LGTM.  I'm going to merge this but may back it out if the boxes go flakey on the left side of the page.

**Comment by TomSweeneyRedHat** [comment]
And it worked!  Thx @rhatdan

**Comment by TomSweeneyRedHat** [comment]
Yes it does, still a couple of nits that @sanjabonic is chasing.  1) if you click on the yellow box and not the word in the box you don't go anywhere.  2) If you shrink your web browser's height, so you only see the first couple of options in the column, you can't get to the option (no scroll bar).  Not an issue now, but might be if we add more links there.  These are css issues that sanja has way more experience than I.

I've also a list of todos to get to, but it's really slick overall.  I want to try and play with having the Buildah black/white cartoon super hero dog as the background of the main page.  Not sure if it would be too busy or not like that.

**Comment by rhatdan** [comment]
Also we need to see if there is a way to test these changes without committing.

**Comment by TomSweeneyRedHat** [comment]
Step ahead of you for once @rhatdan, I pinged @sanjabonic about that earlier today.  Adding blogs/releases/other posts aren't a problem, but it would be nice to be able to see a preview of a change like this one before merging.

**Pull request merged**